README.md

```diff
@@ -266,6 +266,8 @@ Here is an example:
         title = "Android Studio Sync"
         # Measure an Android studio sync
         android-studio-sync {
+            # Enable cleaning ide cache before very sync
+            # clean-ide-cache-before-sync = false
```

Comment:
Unfortunately without a value it does not work. I think changing it to `first-sync` makes sense, so I used that option.

We could also change it to a mutator and use it like `clear-ide-cache-before = BUILD`, but it would require more work, since we would need to share the client between mutator and then actual execution or at least somehow communicate to a build process that is should restart. I think this is currently not required and it will complicate things a bit, so maybe I should not make it a mutator atm. WDYT?

Reply:
I don't understand why it needs to be a mutator when using the `clear-ide-cache-before = BUILD` syntax. One is the DSL, the other is the implementation. For example, you could say the Android Studio sync to do one invalidate caches at the beginning, and then only do a regular sync afterwards.